new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/Operator.java

```diff
@@ -3,35 +3,73 @@
 
 package oracle.weblogic.kubernetes.actions.impl;
 
+import io.kubernetes.client.openapi.ApiException;
 import oracle.weblogic.kubernetes.actions.impl.primitive.Helm;
+import oracle.weblogic.kubernetes.extensions.LoggedTest;
 
-public class Operator {
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class Operator implements LoggedTest {
 
   /**
    * The URL of the Operator's Helm Repository.
    */
+  //Question - is this URL correct?
   private static String OPERATOR_HELM_REPO_URL = "https://oracle.github.io/weblogic-kubernetes-operator/charts";
+
   /**
    * The name of the Operator's Helm Chart (in the repository).
    */
   private static String OPERATOR_CHART_NAME = "weblogic-operator/weblogic-operator";
 
+  /**
+   * Install WebLogic Kubernetes Operator.
+   *
+   * @param params operator parameters for helm values
+   * @return true if the operator is successfully installed, false otherwise.
+   * @throws ApiException - if Kubernetes client API call fails
+   */
+  public static boolean install(OperatorParams params) throws ApiException {
+
+    String namespace = params.getNamespace();
+    String serviceAccount = params.getServiceAccount();
+
+    // assertions for required parameters
+    assertThat(namespace)
+        .as("make sure namespace is not empty or null")
+        .isNotNull()
+        .isNotEmpty();
+
+    assertThat(params.getReleaseName())
+        .as("make sure namespace is not empty or null")
+        .isNotNull()
+        .isNotEmpty();
+
+    // create namespace
+    if (!Namespace.exists(namespace)) {
```

Comment:
again, this is surprising/unexpected.  the method is installOperator - i would not expect that to create a namespace. if you try to deploy something into a non-existent namespace in k8s, it will give an error.  that is what i would expect this to do too. we should try to stick to the standard/predictable/expected behavior

Reply:
ok ..I will create the namespace, service account in the test class ..